teps/0104-tasklevel-resource-requirements.md

```diff
@@ -125,17 +122,17 @@ spec:
 
 This field should also be added to [PipelineRun.TaskRunSpecs][taskRunSpecs].
```

Comment:
think we still need this field under `PipelineRun` right? 

there is no "legacy" `Resources` field, but will keep using `ComputeResources` for a consistent naming

Reply:
yes, it still makes sense to have this in PipelineRun.Spec.TaskRunSpecs under "computeResources"